File: INICIANTE/1037.py
#1037 - Intervalo
num = float(input())

if num>=0 and num<=100:
    if num<=25.0:
        print('Intervalo [0,25]')
    elif num<=50.0:
        print('Intervalo (25,50]')
    elif num<=75.0:
        print('Intervalo (50,75]')
    elif num<=100.0:
        print('Intervalo (75,100]')
else:
    print('Fora de intervalo')

File: INICIANTE/1037.cpp
#include <iostream>

using namespace std;

int main() {

    double a;

    cin>>a;
    if(a<0.0000 || a>100.0000)
        cout<<"Fora de intervalo"<<endl;
    else if(a>=0.0000 && a<=25.0000)
        cout<<"Intervalo [0,25]"<<endl;
    else if(a>25.0000 && a<=50.0000)
        cout<<"Intervalo (25,50]"<<endl;
    else if(a>50.0000 && a<=75.0000)
        cout<<"Intervalo (50,75]"<<endl;
    else if(a>75.0000 || a<=100.0000)
        cout<<"Intervalo (75,100]"<<endl;

    return 0;
}
